resources/views/mail/travel/allassignmentscomplete.blade.php

```diff
@@ -1,13 +1,29 @@
 Hi {{ $travel->primaryContact->preferred_first_name }},
-
-All travelers have paid the travel fee{{ $travel->tar_required ? ' and submitted Travel Authority Requests' : '' }} for {{ $travel->name }}. Contact the treasurer to book travel.
+{{-- all assignments paid and documented --}}
+@if($travel->assignments()->unpaid()->doesntExist() && $travel->assignments()->needDocuSign()->doesntExist())
+All travelers have paid the travel fee{{ $travel->tar_required ? ' and submitted travel forms' : '' }} for {{ $travel->name }}. Contact the treasurer to book travel.
 @if($travel->tar_required)
 
-You can download all submitted forms from {{ config('app.name') }} at {{ route('nova.pages.detail', ['resource' => 'travel', 'resourceId' => $travel->id]) }}, under the action menu (three dots in top right).
+You can download all submitted forms from {{ config('app.name') }} at {{ route('nova.pages.detail', ['resource' => \App\Nova\Travel::uriKey(), 'resourceId' => $travel->id]) }}, under the action menu (three dots in top right).
 @endif
 
 Bon voyage!
+{{-- all assignments documented, not paid--}}
+@elseif($travel->assignments()->unpaid()->exists() && $travel->assignments()->needDocuSign()->doesntExist())
```

Comment:
Seems like you use these conditions quite a bit, might be worth adding a couple extra methods on the model to make the logic easier to follow